packages/blade/src/components/Input/PasswordInput/PasswordInput.stories.tsx

```diff
@@ -15,6 +15,7 @@ const Page = (): ReactElement => {
   return (
     <StoryPageWrapper
       componentName="PasswordInput"
+      apiDecisionLink="https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Input/PasswordInput/_decisions/decisions.md"
```

Comment:
Currently, it opens API descision on GitHub, can we not host it on storybook itself?

Reply:
A bit difficult to do in current setup (I am not sure if it will be able to resolve locally referenced images correctly). 

Also the alignments on storybook are a mess when used with markdown or mdx so need to fix that before doing this 😢 .